rfc/20250317-ephemeral-resources.md

```diff
@@ -220,7 +220,7 @@ Locals marked as ephemeral are available during plan and apply phase and can be
 * `connection` and `provisioner` blocks
 
 #### Ephemeral resource
-In contrast with the write-only arguments where only specifically tagged attributes are not stored in the state/plan file, `ephemeral` resources are not stored entirely.
+In contrast with the write-only arguments where only specifically tagged attributes are not stored in the state/plan file, `ephemeral` resources must not be stored in the state file and only partially stored in the plan one.
```

Comment:
"only partially stored" could be clarified to something like "only a reference stored"

Reply:
Fixed in 2d39a802e98534f2030983a957fb46154b73e825.
Thanks for the suggestion.

Reply:
> I was going to ask: what "**partially** stored in plan" means? (not sure if this was clarified by earlier PRs)
> 
> edit: answered below

I specifically didn't add more details here, because when I wrote the RFC, I had a rough time separating details from big picture. I just want to keep things straight forward in this section and have a more in-depth analysis in the technical section.